plugins/serializers/extr/README.md

```diff
@@ -50,16 +55,20 @@ The `extr` output data format converts metrics into JSON documents, performing t
 The following Telegraf batched metrics
    
 ```text
-CpuStats,serialnum=XYZ-1234 core_key=0i,usage_min=35.1,usage_max=99.1,usage_avg=35.1
-CpuStats,serialnum=XYZ-1234 core_key=1i,usage_min=50.1,usage_max=88.1,usage_avg=51.1
-FanStats,serialnum=XYZ-1234 slot_key=1i,tray_key=2i,fan_key=10i,rpm_min=4101,rpm_max=5001,rpm_avg=4201,pwm_min=31,pwm_max=41,pwm_avg=31
-FanStats,serialnum=XYZ-1234 slot_key=1i,tray_key=2i,fan_key=11i,rpm_min=4001,rpm_max=4991,rpm_avg=4001,pwm_min=41,pwm_max=51,pwm_avg=41
-FanStats,serialnum=XYZ-1234 slot_key=2i,tray_key=3i,fan_key=9i,rpm_min=2101,rpm_max=3211,rpm_avg=2201,pwm_min=11,pwm_max=41,pwm_avg=11
-CpuStats,serialnum=XYZ-1234 core_key=0i,usage_min=10.2,usage_max=91.2,usage_avg=44.2
-CpuStats,serialnum=XYZ-1234 core_key=1i,usage_min=22.2,usage_max=89.2,usage_avg=41.2
-CpuStats,serialnum=XYZ-1234 core_key=2i,usage_min=33.2,usage_max=79.2,usage_avg=47.2
-FanStats,serialnum=XYZ-1234 slot_key=1i,tray_key=2i,fan_key=10i,rpm_min=4112,rpm_max=5012,rpm_avg=4212,pwm_min=32,pwm_max=52,pwm_avg=32
-FanStats,serialnum=XYZ-1234 slot_key=1i,tray_key=2i,fan_key=11i,rpm_min=5002,rpm_max=5092,rpm_avg=4102,pwm_min=52,pwm_max=62,pwm_avg=52
+CpuStats,serialnumber=XYZ-1234 core_key=0i,usage_min=35.1,usage_max=99.1,usage_avg=35.1
+CpuStats,serialnumber=XYZ-1234 core_key=1i,usage_min=50.1,usage_max=88.1,usage_avg=51.1
+FanStateChanged,serialnumber=XYZ-1234 slot_key=2i,tray_key=4i,fan_key=1i,partNumber_tag="1647G-00129 800751-00-01",revision_tag="01",airFlowDirection_tag="FrontToBack",state_old="Ok",state_new="Failed"
+FanStateChanged,serialnumber=XYZ-1234 slot_key=2i,tray_key=4i,fan_key=2i,partNumber_tag="1647G-00129 800751-00-01",revision_tag="01",airFlowDirection_tag="FrontToBack",state_old="Failed",state_new="Ok"
+FanStats,serialnumber=XYZ-1234 slot_key=1i,tray_key=2i,fan_key=10i,rpm_min=4101,rpm_max=5001,rpm_avg=4201,pwm_min=31,pwm_max=41,pwm_avg=31
```

Comment:
slot_key is removed now. In lines 62, 63 and 64.

Reply:
I'll fix.  Note, this is just example output, so not a big deal.

Reply:
Fixed